Geometry_Engine/Create/NurbsCurve.cs

```diff
@@ -49,11 +49,10 @@ public static NurbsCurve NurbsCurve(IEnumerable<Point> controlPoints, IEnumerabl
         }
 
         /***************************************************/
-
-        [NotImplemented]
+                
         public static NurbsCurve NurbsCurve(IEnumerable<Point> controlPoints, IEnumerable<double> weights, IEnumerable<double> knots)
```

Comment:
I think that thanks to https://github.com/BHoM/BHoM_UI/pull/147 we can also remove this method.

Reply:
Good point. Two things:
1. I understand that it will require change in creating Nurbs in the code (from `Create.NurbsCurve(pts, knots, weights)` to `new NurbsCurve { ControlPoints = pts, Weights = weights, Knots = knots }`). I'm totally okay with it but as far as I know those two ways are implemented alternately. Does it mean that from now we are going to remove most of the Create methods from the engine and use direct constructors?
2. If so I think that the whole `Create.Nurbsucrve.cs` could be deleted as all of the methods there are NotImplemented. I don't know if it would be better to do it now or just left it as NotImplemented and wait until some major clean-up.

Reply:
Hey @LMarkowski , I think that in the light of https://github.com/BHoM/BHoM/pull/603#issuecomment-551840486 it's good to keep this methods as it is. If we implement `IImmutable`, we will need to explicitly have a `Create` method.